log/log.go

```diff
@@ -39,6 +59,18 @@ func GetOSSigChannel() chan os.Signal {
 	return c
 }
 
+func RegexCompile(o Options) {
+
+	CNamespace, _ = regexp.Compile(o.Namespace)
+	CLogtype, _ = regexp.Compile("(?i)" + o.LogType)
+	COperation, _ = regexp.Compile("(?i)" + o.Operation)
+	CContainerName, _ = regexp.Compile(o.ContainerName)
+	CPodName, _ = regexp.Compile(o.PodName)
```

Comment:
Pod, Container and Namespace can be case insensitive too, Refer https://github.com/kubearmor/kubearmor-client/pull/52#pullrequestreview-921879220